### PR TITLE
Cleanup ibft logging levels

### PR DESCRIPTION
**Pull request by jframe**
## PR description
Updating IBFT logging levels to make more consistent.
Debug level is now predominately about the state machine send, receives and quorums.
Info now also includes validation failures so that is visible.
Trace now alsol includes additional receives in parts of the state machine

### CODE REVIEW

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java [hunk 1]
```diff
@@ -84,13 +84,7 @@ public ConsensusRoundIdentifier getRoundIdentifier() {
...
   public void createAndSendProposalMessage(final long headerTimeStampSeconds) {
     final Block block = blockCreator.createBlock(headerTimeStampSeconds);
-    final IbftExtraData extraData = IbftExtraData.decode(block.getHeader().getExtraData());
-    LOG.debug(
-        "Creating proposed block. round={} extraData={} blockHeader={}",
-        roundState.getRoundIdentifier(),
-        extraData,
-        block.getHeader());
-
+    LOG.debug("Creating proposed block. round={}", roundState.getRoundIdentifier());
```
Comment (rain-on) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java]: nit: tempted to put a trace in here too with the additional info?

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java [hunk 2]
```diff
@@ -94,7 +94,7 @@ public RoundChangeManager(
   public Optional<Collection<RoundChange>> appendRoundChangeMessage(final RoundChange msg) {
 
     if (!isMessageValid(msg)) {
-      LOG.info("RoundChange message was invalid.");
+      LOG.debug("RoundChange message was invalid.");
```
Comment (rain-on) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java]: Why is an invalid RoundChange debug, but an invalid Proposal an info? (see IbftBlockHeightManager) - I think I prefer info in both places (this is an exceptional behaviour and we want to see it even normally)

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java [hunk 3]
```diff
@@ -80,15 +80,13 @@ public boolean setProposedBlock(final Proposal msg) {
   public void addPrepareMessage(final Prepare msg) {
     if (!proposalMessage.isPresent() || validator.validatePrepare(msg)) {
       prepareMessages.add(msg);
-      LOG.debug("Round state added prepare message prepare={}", msg);
```
Comment (rain-on) [inline consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java]: nit: not even trace? ... maybe not ... (just want to confirm!)

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java (hunk 1 above)
Comment (jframe) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRound.java]: done. Didn't it was useful at debug level and it made the logs difficult to read. but at trace it's probably appropriate.

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java (hunk 2 above)
Comment (jframe) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java]: done

File: consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java (hunk 3 above)
Comment (jframe) [reply consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java]: done. made it trace

**Pull request merged**